.gitattributes

```diff
@@ -2,6 +2,9 @@
 mock_*.go linguist-generated
 *_mock.go linguist-generated
 
+# genny files
```

Comment:
eventually I want to fix how we generate genny files to include DO NOT EDIT, which should automatically be excluded from github. for now this quick solution works.